serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -9,9 +9,9 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-cudnn8-devel-ubuntu22.04
+ARG version=12.3.2-devel-ubuntu22.04
```

Comment:
Can we use cu12.4.1?

Reply:
Yea i think that should be fine as well - will update